fastkml/kml.py

```diff
@@ -1220,16 +1220,27 @@ def rotation(self, value):
         else:
             raise ValueError
 
-    def latLonBox(self, north, south, east, west, rotation=0):
-        # TODO: Check for bounds (0:+/-90 or 0:+/-180 degrees)
+    def LatLonBox(self, north, south, east, west, rotation=0):
```

Comment:
the project has mixed `camelCase` and `snake_case` names, not sure what I should default to. Specially in cases like ViewVolume and ImagePyramid that are attributes and do not have they're own class

Reply:
The camelCase names are legacy/technical debt, new code should use pep8 snake_case